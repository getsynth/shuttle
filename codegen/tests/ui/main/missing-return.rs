#[shuttle_codegen::main]
async fn missing_return() {}
