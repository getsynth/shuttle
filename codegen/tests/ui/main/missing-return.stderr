error: shuttle_runtime::main functions need to return a service

         = help: See the docs for services with first class support
         = note: https://docs.rs/shuttle-service/latest/shuttle_runtime/attr.main.html#shuttle-supported-services

 --> tests/ui/main/missing-return.rs:2:1
  |
2 | async fn missing_return() {}
  | ^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/main/missing-return.rs:2:29
  |
2 | async fn missing_return() {}
  |                             ^ consider adding a `main` function to `$DIR/tests/ui/main/missing-return.rs`
